### DISCUSSION TITLE
DAI 2024-2025 - Check out my GitHub profile! - Sara Camassa

### DISCUSSION THREAD

**Discussion by scamassa** [heig-vd-dai-course/heig-vd-dai-course#245] [Show and tell]
https://github.com/scamassa/scamassa

**Comment by ludelafo** [comment]
Simple yet elegant GitHub profile! What have you done in Vue.js? :)

**Reply by scamassa** [reply]
I had a short internship where I had to create a website to handle tennis courts reservations, and I had to use Vue.js to do it ! I had never even used javascript before but it was fun to learn

**Reply by ludelafo** [reply]
Oh that's cool! In a future course you should do some React as well, hope you'll enjoy it as well!
